src/muxer.ts

```diff
@@ -148,7 +148,7 @@ export class Muxer<T extends Target> {
 				throw new Error(`Unsupported video codec: ${options.video.codec}`);
 			}
 
-			if (options.video.rotation !== undefined && ![0, 90, 180, 270].includes(options.video.rotation)) {
+			if (typeof options.video.rotation === 'number' && ![0, 90, 180, 270].includes(options.video.rotation)) {
 				throw new Error(`Invalid video rotation: ${options.video.rotation}. Has to be 0, 90, 180 or 270.`);
 			}
```

Comment:
You should also verify the transformation matrix if it's passed in. We should ensure it's an array of exactly 9 numbers.

Reply:
Added an explicit check for size and element type of the array.  

In general I am not a fan of checking if a library consumer violates the API contract.